survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
```

Comment:
We could just target a release? 
https://github.com/kgorskowski/ebs_bckup/releases/tag/v0.3